### PR TITLE
#156991533 Integrate Coveralls

### PR DESCRIPTION
**Pull request by zenjutahi**
#### What does this PR do?
Integrate Coveralls

#### Description of Task to be completed?
Set up the necessary configuration for the coverage data of the codebase using Coveralls
Copied the markdown for the coverage and added it to read me

#### How should this be manually tested?
Every push has to pass through Coveralls and the coverage viewed on the badge

#### What are the relevant pivotal tracker stories?
#156991533

### CODE REVIEW

**Review by jimmykamau — CHANGES_REQUESTED** [review]
Did you branch off `develop` or the branch you had been previously working on?

**Comment by zenjutahi** [comment]
I branched from my local `develop`, not the same branch i did circleci on, i just added the necessary file configurations to ensure it passes

**Comment by zenjutahi** [comment]
resolved conflicts and unnecessary changes in unexpected files

**Review by Bonifase — APPROVED** [review]
The build is fine, can be merged to develop

**Comment by murageden** [comment]
The task is complete and build is successfull and can be merged

**Review by Collins33 — APPROVED** [review]

**Review by ghost — APPROVED** [review]
The build is successful. Coveralls badge is showing test coverage okay

**Comment by zenjutahi** [comment]
I changed coveralls.yml

**Review by alexmochu — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+[![Coverage Status](https://coveralls.io/repos/github/andela/hc-bulls/badge.svg?branch=ft-Integrate-Coveralls%23156991533)](https://coveralls.io/github/andela/hc-bulls?branch=ft-Integrate-Coveralls%23156991533)
```
Comment (jimmykamau) [inline README.md]: Point the branch to Develop

**Comment by zenjutahi** [comment]
revolved conflict in README.md and setup coveralls badge.

**Review by jimmykamau — APPROVED** [review]

**Pull request merged**